switchmap/poller/snmp/mib/cisco/mib_ciscoprocess.py

```
@@ -0,0 +1,142 @@
+        # Process OID - Enhanced memory pool free (high capacity)
+        oid = ".1.3.6.1.4.1.9.9.48.1.1.1.6"
+
+        # Return OID value for unittests
+        if oidonly is True:
+            return oid
+
+        try:
+            results = await self.snmp_object.swalk(oid, normalized=True)
+            free_memory = sum(results.values())
+
+            return free_memory
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)